v2/datastream-common/src/main/java/com/google/cloud/teleport/v2/datastream/sources/DataStreamIO.java

```diff
@@ -172,6 +174,12 @@ public DataStreamIO withDirectoryWatchDuration(Duration directoryWatchDuration)
     return this;
   }
 
+  /** Set applyReshuffle to false inorder to skip the reshuffle step on datastream records. */
```

Comment:
Add more docs here and when this should be used and why is this being added.

Reply:
done